app/components/Settings/scheduler/scheduler.jsx

```diff
@@ -255,6 +255,20 @@ class Scheduler extends Component {
                         </button>
                     )}
                 </Row>
+                {!loggedIn && (
+                    <Row
+                        style={{
+                            display: 'box',
+                            borderLeft: '4px solid red',
+                            padding: '6px',
+                            margin: '24px 0 24px 20px',
+                            justifyContent: 'flex-start'
+                        }}
+                    >
+                        <b>Scheduled queries will not run unless you are logged in.</b> Click <a href="/login">here</a>{' '}
```

Comment:
Perhaps I'm misunderstand the behaviour here... If the Plot.ly tab says I'm "not logged in" it's still possible for the scheduled queries to run?

Reply:
The scheduled queries are run by the backend. The backend stores the user's access token, so that it can run the scheduled queries in `~/.plotly/connector/setting.yaml` even if the user isn't logged in (e.g. after restating Falcon).

Reply:
OK awesome. So what does logging in do then? If the backend has my token, and I'm "logged out" why must I log in to create a scheduled query?

Reply:
Ah, I (more or less) understand, thanks!

Reply:
To be clear, this warning should be removed then, correct? 🙂

Reply:
correct.